editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -53,7 +53,6 @@
 #include "editor/plugins/gizmos/cpu_particles_3d_gizmo_plugin.h"
 #include "editor/plugins/gizmos/decal_gizmo_plugin.h"
 #include "editor/plugins/gizmos/fog_volume_gizmo_plugin.h"
-#include "editor/plugins/gizmos/geometry_instance_3d_gizmo_plugin.h"
```

Comment:
You removed this include and it's not indirectly included

Reply:
Thanks for investigating! Should be fixed now